politeiawww/backend.go

```diff
@@ -244,6 +244,11 @@ func (b *backend) _convertPropFromInventoryRecord(r inventoryRecord) www.Proposa
 	// Set the comments num.
 	proposal.NumComments = uint(len(r.comments))
 
+	// Set the proposals status timestamp
```

Comment:
Should this be `Set the proposals status timestamps` since it is setting multiple timestamps?